pkg/analytics/track.go

```diff
@@ -57,8 +57,8 @@ const (
 	previewDestroyEvent      = "DestroyPreview"
 	execEvent                = "Exec"
 	signupEvent              = "Signup"
-	contextEvent             = "Context"
-	contextUseNamespaceEvent = "Context Use-namespace"
+	contextEvent             = "GetContext"
```

Comment:
```suggestion
	contextEvent             = "Context"
```

Reply:
This change scares me a lot . We are changing the event we are sending to mixpannel. Are we ok with that

Reply:
No, that's why I'm doing the suggestion, because this PR changes it. All my suggestions are to get the original value

Reply:
good catch guys, I was still pending a self-review but thanks for spotting it!